code/modules/clothing/gloves/color.dm

```diff
@@ -5,19 +5,19 @@
 	siemens_coefficient = 0
 	permeability_coefficient = 0.05
 
-/obj/item/clothing/gloves/fyellow                             //Cheap Chinese Crap
-	desc = "These gloves are cheap copies of proper insulated gloves. No way this can end badly."
-	name = "budget insulated gloves"
+/obj/item/clothing/gloves/fyellow                             //VORE:These used to be budget insuls. Not changing the path due to upstream conflicts.
+	name = "yelllow gloves"

```

Comment:
This, on the other hand, does. As well as the below line. //VOREStation Edit
Also spelling error.